src/sonic_ax_impl/mibs/ietf/fan_sensor_data.py

```diff
@@ -0,0 +1,92 @@
+import re
+
+from .physical_entity_sub_oid_generator import SENSOR_TYPE_FAN
+
+
+class FANSensorData:
```

Comment:
How about add a BaseSensorData class? FANSensorData, PSUSensorData and ThermalSensorData can share implementation in BaseSensorData.

Reply:
updated